portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -4729,6 +4729,268 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-180198 TC-1 and TC-4: The user could be assigned to an existing user group at Portal, during login using Okta as SAML IDP (AC-1), and the missing user group is not created at Portal."
+	@priority = 5
+	test ViewSPInitiatedSSOWithOktaAndUserGroups {
+		property app.server.types = "jboss,tcserver,tomcat,wildfly";
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}feature.flag.LPS-180198=true";
+		property database.types = "hypersonic,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
+		property portal.ssl.enabled = "true";
+		property test.name.skip.portal.instance = "SAML#ViewSPInitiatedSSOWithOktaAndUserGroups";
```

Comment:
Need to quarantine test case in release run.
property portal.release = "false";
The same changes are needed for other new test cases.

Reply:
Thank you Joyce, I nearly forgot this!